src/serina.ts

```diff
@@ -6,7 +6,7 @@ import Year from 'filters/year/year';
 import Time from 'filters/time/time';
 import Dates from 'filters/dates/dates';
 import PartialDates from './filters/dates/partialDates';
-import DateAndTime from './filters/dates/dateAndTime';
+import DateAndTime from './filters/dates/dateAndTime/dateAndTime';
```

Comment:
Can we add another test case in serina.test.ts for the new filter?

Reply:
sure